go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/kserve/modelmesh-serving
 
-go 1.21
+go 1.22.0
+
+toolchain go1.24.2
```

Comment:
This is requesting go 1.24. I don't think it will build correctly.

Reply:
updated